README.md

```diff
@@ -227,7 +257,7 @@ Scanner vendors are supposed to implement Scanner Adapters exposing the Scanner
 
 - The `Accept` request header is required to indicate to Scanner Adapter an intended scan report format
 - If the client does not specify the `Accept` header it's assumed to be Harbor vulnerability report with the
-  MIME type `application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0`.
+  MIME type `application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0`. This behavior is currently maintained only for backward compatibility purposes.
```

Comment:
@prahaladdarkin 

Would you please also update the compatibility list doc link to the latest one `https://goharbor.io/docs/edge/install-config/harbor-compatibility-list/`?

Reply:
And please also add the new scanner vendors to the list that is at the bottom of this doc.

[Sysdig](https://github.com/sysdiglabs/harbor-scanner-sysdig-secure)

[TensorSecurity](https://github.com/tensorsecurity/harbor-scanner)

Reply:
Done. 
Just to confirm - would Sysdig and TensorSecurity confirm to the version 1.1 of the schema or the 1.0? If it is 1.0 I would  have to update the README.md on the 1.0 branch